backend/app/api/v1/endpoints/entities.py

```
@@ -84,29 +93,216 @@ async def _run_extraction_task(
+            # Only cache when the LLM extraction succeeded so transient errors
+            # don't poison the cache with empty results.
+            if extraction_succeeded:
+                try:
+                    await cache_set(
+                        ent_cache_key,
+                        ent.model_dump(),
+                        ttl_seconds=EXTRACTION_CHUNK_CACHE_TTL,
+                    )
+                except Exception:
+                    # Cache failures should never fail extraction
+                    pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)